charts/fybrik/values.yaml

```diff
@@ -66,16 +66,14 @@ coordinator:
   catalog: "katalog"
 
   # Overrides the catalog connector URL.
-  # Defaults to `<catalog>-connector:80`.
```

Comment:
Keeping a comment would be useful. Like `Defaults to `http://<catalog>-connector:80`.

Reply:
Done. Thanks